api-docs/openapi-diff.ts

```diff
@@ -0,0 +1,37 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+/* eslint import/no-unresolved:0 */
+
+const fs = require('fs');
+const yaml = require('js-yaml');
+const jsonDiff = require('json-diff');
+const https = require('https');
+
+async function main() {
+  const request = new Promise((resolve, reject) =>
+    https.get(
+      `https://raw.githubusercontent.com/getsentry/sentry-api-schema/main/openapi-derefed.json`,
+      res => {
+        res.setEncoding('utf8');
+        let rawData = '';
+        res.on('data', chunk => (rawData += chunk));
+        res.on('end', () => {
+          try {
+            resolve(JSON.parse(rawData));
+          } catch (e) {
+            reject(e.message);
+          }
+        });
+      }
+    )
+  );
+
+  const openApiData = await request;
```

Comment:
If you're going to write like this, let's just do
```suggestion
  const openApiData = await new Promise((resolve, reject) =>
    https.get(
      `https://raw.githubusercontent.com/getsentry/sentry-api-schema/main/openapi-derefed.json`,
      res => {
        res.setEncoding('utf8');
        let rawData = '';
        res.on('data', chunk => (rawData += chunk));
        res.on('end', () => {
          try {
            resolve(JSON.parse(rawData));
          } catch (e) {
            reject(e.message);
          }
        });
      }
    )
  );
```